doc/release-notes.md

```diff
@@ -59,6 +59,15 @@ Notable changes
 P2P and network changes
 -----------------------
 
+- This release removes support for Tor version 2 hidden services in favor of Tor
+  v3 only, as the Tor network [dropped support for Tor
+  v2](https://blog.torproject.org/v2-deprecation-timeline) with the release of
+  Tor version 0.4.6.  Henceforth, Bitcoin Core ignores Tor v2 addresses; it
+  neither rumors them over the network to other peers nor stores them in memory
+  or to `peers.dat`.  Tor v2 addresses are ignored by
+  `contrib/seeds/generate-seeds.py` and have been removed from the hard-coded
```

Comment:
I don't think changes to maintainer-facing tools such as `generate-seeds` need to be in the release notes. Not a big deal though.

Reply:
Good point, removed.

FWIW if #22179 is merged, I planned to append it to this entry manually in the wiki.